src/ops/constant.rs

```diff
@@ -63,20 +131,11 @@ pub(crate) const HUGR_MAX_INT_WIDTH: HugrIntWidthStore =
 #[allow(missing_docs)]
 pub enum ConstValue {
     /// An arbitrary length integer constant.
-    Int {
-        value: HugrIntValueStore,
-        width: HugrIntWidthStore,
-    },
+    Int(HugrIntValueStore),
     /// Double precision float
     F64(f64),
     /// A constant specifying a variant of a Sum type.
-    Sum {
-        tag: usize,
-        // We require the type to be entirely Classic (i.e. we don't allow
-        // a classic variant of a Sum with other variants that are linear)
-        variants: ClassicRow,
-        val: Box<ConstValue>,
-    },
+    Sum { tag: usize, val: Box<ConstValue> },
```

Comment:
I'd be tempted to make this a tuple. The types will ensure you don't get them the wrong way around and it's a lot easier on the eyes/fingers. Up to you!